### ISSUE TITLE
Can we add an option to put the screen controller outside of the video player?

### ISSUE THREAD

**Issue by bxacsq** [iina/iina#2717]
**What you want IINA to do:**
Can we add an option to put the screen controller outside of the video player?

**What IINA does currently:**
Right now, the 3 layouts for the screen controller always put it on the video player.

**Why you think this should be added:**
When I need to replay some short episodes, the hovered screen controller bothers me a lot.

**Examples of other projects that have something similar:**
Like VLC does:
<img width="606" alt="Screen Shot 2020-01-13 at 16 02 13" src="https://user-images.githubusercontent.com/18612853/72240152-3a1b2e00-361e-11ea-9fab-ffed17b3638c.png">

**Comment by jerkstore369** [comment]
Yes, I second this. It would be better if the title bar and video controls bar were outside of the video image itself. If it does get implemented it'll take like a year with the way development is going...

**Comment by hazarek** [comment]
also quick settings panel

**Comment by saagarjha** [comment]
@hazarek I think moving out the Quick Settings panel is #42.

**Comment by hazarek** [comment]
> @hazarek I think moving out the Quick Settings panel is #42.

oh sorry, thanks.

**Comment by tonglil** [comment]
Also see https://github.com/iina/iina/issues/1340

**Comment by uiryuu** [comment]
Fixed in #6132.

**Comment by uiryuu** [comment]
This feature has been partially implemented in the `develop` branch and will be included in the next major release of IINA (v1.5.0). Further improvements, adjustments, and bug fixes related to this issue are expected. If you're interested, you can try it out by downloading a [Nightly build](https://iina.io/nightly/) (a GitHub account is required).
